fakes/bar.go

```diff
@@ -0,0 +1,17 @@
+package fakes
```

Comment:
yeah, @vito I considered this but the fakes here are much smaller. It was really automate versus smaller fakes - I leaned toward having tiny ones.